spring-test/src/main/java/org/springframework/test/context/cache/ContextCache.java

```diff
@@ -155,7 +155,7 @@ public interface ContextCache {
 	void clear();
 
 	/**
-	 * Clear hit and miss count statistics for the cache (i.e., reset counters to zero).
+	 * Clear hit-and-miss count statistics for the cache (i.e., reset counters to zero).
```

Comment:
It's truly not a [hit-and-miss](https://www.merriam-webster.com/dictionary/hit-and-miss) scenario. 🤣 

Rather, those are statistics for "hit counts" and "miss counts".

I'll address this while merging.